<launch>
    <arg name="engine_name" default="$(find yolov5_detect)/weights/bluecar_1024_n_2070.engine"/>
    <!-- <arg name="video_name" default="$(find yolov5_detect)/detect/test_vedio/1.jpeg"/> -->
    <arg name="video_name" default="rtsp://admin:admin@192.168.42.108:554/cam/realmonitor?channel=1&amp;subtype=2"/>
    <arg name="video_out_path" default="$(find yolov5_detect)/detect/"/>
    <arg name="sort_p" default="2"/>

    <node pkg="yolov5_detect" type="yolov5_detect_fly" name="yolov5_detect_node" output="screen">
        <param name="node_num" type = "string" value = "uav1"/>
        <param name="engine_name" type = "string" value = "$(arg engine_name)"/>	
        <param name="video_name" type = "string" value = "$(arg video_name)"/> 
        <rosparam param="img_list">[1920, 1080, 1024, 1024, 2, 1, 128, 50]</rosparam>
        <!-- <rosparam param="img_list">[4096, 2160, 1024, 1024, 4, 2, 90, 50]</rosparam> -->
        <param name="video_out_path" type = "string" value = "$(arg video_out_path)"/> 
        <param name="sort_p" type = "int" value = "$(arg sort_p)"/> 
    </node>
</launch>



